src/Repository/EventRepository.php

```diff
@@ -512,6 +512,20 @@ public function countReferentEventsInReferentManagedArea(Adherent $referent): ar
         return $this->aggregateCountByMonth($result, BaseEvent::REFERENT_EVENT_TYPE.'s');
     }
 
+    public function countTotalEventsInReferentManagedAreaForCurrentMonth(Adherent $referent): int
+    {
+        $this->checkReferent($referent);
+
+        $query = $this->queryCountByMonth($referent, 1);
+
+        $result = $query
+            ->getQuery()
+            ->getSingleResult()
+        ;
+
+        return (int) $result['count'];
```

Comment:
`return (int) $....->getSingleScalarResult();` ?

Reply:
It will not work, because there are more than one column `count`.